src/module/module-collection.js

```diff
@@ -49,7 +49,7 @@ export default class ModuleCollection {
   unregister (path) {
     const parent = this.get(path.slice(0, -1))
     const key = path[path.length - 1]
-    if (!parent.getChild(key).runtime) return
+    if (!parent.getChild(key) || !parent.getChild(key).runtime) return
```

Comment:
I prefer assertion rather than silently ignoring when the specified module does not exist. Can we change the implementation to that?

Reply:
I agree, maybe you can do it better latter